test/parallel/test-https-drain.js

```diff
@@ -21,17 +21,18 @@
 
 'use strict';
 const common = require('../common');
+const fixtures = require('../common/fixtures');
+
 if (!common.hasCrypto)
   common.skip('missing crypto');
 
 const assert = require('assert');
 const https = require('https');
 const fs = require('fs');
-const path = require('path');
 
 const options = {
-  key: fs.readFileSync(path.join(common.fixturesDir, 'test_key.pem')),
-  cert: fs.readFileSync(path.join(common.fixturesDir, 'test_cert.pem'))
+  key: fs.readFileSync(fixtures.path('test_key.pem')),
```

Comment:
This could be cleaned up further by making use of [`fixtures.readSync()`](https://github.com/nodejs/node/tree/master/test/common#fixturesreadsyncargs-enc) to replace the `fs.readFileSuync` _and_ the `path.join`!

Something like `key: fixtures.readSync('test_key.pem'),`